### PR TITLE
Dan newbuild

### PR DESCRIPTION
**Pull request by danparisd**
this moves the build to travis. Cleans up some old build scripts and includes the gemfile.lock so we can get to a known good config if needed.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 <footer class="print-footer" style="display: none;">
   <div class="row">
     <div class="one-half first">
-      <img src="https://cdn.bibleineverylanguage.org/wp-content/uploads/wacs_logo.png" alt="BTT"/>
+      <img src="https://cdn.bibleineverylanguage.org/wp-content/uploads/2018/09/wacs_logo.png" alt="BTT"/>
```
Comment (aunger) [inline _includes/footer.html:21]: Code changes should be in a separate PR from build stuff, but this might not be worth changing.

**Review by aunger — APPROVED** [review]
Dan Newbuild got me $1,000,0000,00000!

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+    secret_access_key:
+      - secure: "iAEwyqFNFgT/kby2XtS8vCQ5eADWCIe+tKekma35TceZp8zDkhyGjSNbvGqwC7Q3OwNrJ65Slceafq5ztzqjpZ6tPUbT66UaV7c6GLCbdnwstADfiU006BhNs9BIeUU0TOQD+9zXd/NWK9zTpkDIWgNe9TeEn+otxmVZxErHfGaugD4yAx1vopuZbbT9KpMTs5hsgBhqOyNPCbYoAv6O9kKaTDCwPqUDgnnBc0Ede/DpGq9SHceywl37qWRGCqmmNias3lwdWKV02rLRUCvdRlXyo4jihxcbJHE4PJ9RtRQcrpIc6obxR0DB0oDC+gbAIKsqGTgSBuNLC5MB35kouKCNXUO56zXofQLgtLnaeLqSJPUSiaEXxz2IW4HJM2vQ+HxD1JcWOyAlQyR8gLlGYiNeebsD47cCg4yTrdc67oZKJpyPeQ8mVz323+lP8m3LXaQqIsfB2zBwAKxP5b7UdcRuOR+0Xyur0lfBjM+qo2YjhvGxZSHn8keEFk6/1xXZdv4Cz9Xpi0w4TLLPaTx0k5mDZi76xHotuCHkxW3/MZF+KYtOA+mWTaSTgoWfHa4vrHI+RAEo/ntzoUaDU97rIC4vcXqTceSkk/eT6vm4cBFxrK2mzk8upP5wD0zm1jlxMFGdiswmQpNunUzDs6+jdLjulPA7RMzVDzKynG5fBN4="
+    bucket: "read.bibletranslationtools.org"
+    edge: true
+    region: us-east-2
+    local_dir: _site
+    skip_cleanup: true
+    on:
```
Comment (aunger) [inline .travis.yml:34]: Consider putting the two "on" blocks next to each other in this file.

Comment (aunger) [reply .travis.yml:34]: (This is minor too, feel free to ignore.)

Comment (danparisd) [reply .travis.yml:34]: What do you mean? It look a little unclear here, but the on blocks belong to separate `- provider` blocks

File: _includes/footer.html (hunk 1 above)
Comment (danparisd) [reply _includes/footer.html:21]: fair. I think it's minor enough we can leave it.

File: .travis.yml (hunk 2 above)
Comment (aunger) [reply .travis.yml:34]: You’re right; I missed a hyphen so thought they were in the same provider. I’m too old for yaml.

**Review by aunger — APPROVED** [review]
LGTM
